coremltools/converters/mil/mil/passes/tests/test_pass_pipeline.py

```diff
@@ -118,6 +118,31 @@ def test_list_available_pipelines(self):
         assert "default" in available_pipelines
         assert "default_palettization" in available_pipelines
 
+    def test_merge_reshape_in_nested_block(self):
```

Comment:
I agree with you that this test in this file should be removed. This testing file focuses on testing the `PassPipeline` infra instead of a specific pass. Could you push another commit to remove this test?

If you need it to run tests locally, maybe only keep the test local but not push to remote. Thanks!